src/test/server/apis/api-router/private-router-test.ts

```diff
@@ -0,0 +1,126 @@
+import anyTest, {TestInterface} from 'ava';
+import * as express from 'express';
+import * as sinon from 'sinon';
+import {SinonSandbox} from 'sinon';
+
+import {PrivateAPIRouter} from '../../../../server/apis/api-router/private-api-router';
+import * as responseHelper from '../../../../server/apis/api-router/response-helper';
+import {userModel} from '../../../../server/models/userModel';
+import {newFakeUserRecord} from '../../../utils/newFakeUserRecord';
+
+type TestContext = {
+  sandbox: SinonSandbox,
+};
+const test = anyTest as TestInterface<TestContext>;
+
+/**
+ * Generates the test context object before each test.
+ */
+test.beforeEach(async (t) => {
+  t.context = {
+    sandbox: sinon.sandbox.create(),
+  };
+});
+
+test.afterEach.always(async (t) => {
+  t.context.sandbox.restore();
+});
+
+test.serial(
```

Comment:
AFAIK, this doesn't need to run in serial. Don't see any singletons that you're potentially using.

Reply:
It's the stubbing that makes this need to be serial